lib/resources/r4/condition.yaml

```diff
@@ -0,0 +1,119 @@
+---
+name: Condition
+field_name_base_url: https://hl7.org/fhir/r4/condition-definitions.html#Condition
+fields:
+- name: clinicalStatus
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: active | recurrence | relapse | inactive | remission | resolved
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.clinicalStatus
+  binding:
+    description: The clinical status of the condition.
+    terminology:
+      - display: Condition Clinical Status Codes
+        system: https://terminology.hl7.org/CodeSystem/condition-clinical
```

Comment:
the system should be http://, the info_link is correct as https://

same for others.

Reply:
Fixed.